xml/System.ComponentModel/DateOnlyConverter.xml

```diff
@@ -57,7 +57,7 @@
       <Docs>
         <param name="context">To be added.</param>
```

Comment:
@tarekgh @eerhardt  - can one of you please provide suggestions with documentation for the params and returns of all the APIs in this file, as well as in TimeOnlyConverter.xml? Otherwise they will keep showing up as undocumented in the report.

Reply:
<param name="context">An <see cref="T:System.ComponentModel.ITypeDescriptorContext" /> that provides a format context.</param>

Reply:
```Xml 
<param name="context">An <see cref="T:System.ComponentModel.ITypeDescriptorContext" /> that provides a format context.</param>
```

Reply:
Thanks. There are many more empty params and returns in this same file. Can you please provide the suggestions using the ` ```suggestion ` snippets ?

Reply:
ok